doc/src/tutorials/arch/multi_mode_ble.rst

```diff
@@ -0,0 +1,303 @@
+         <mux name="mux1" input="ff.Q lut6.out" output="ble6.out">
+           <!-- LUT to output is faster than FF to output on a Stratix IV -->
+           <delay_constant max="25e-12" in_port="lut6.out" out_port="ble6.out"/>
+           <delay_constant max="45e-12" in_port="ff.Q" out_port="ble6.out"/>
+         </mux>
+       </interconnect>
+     </pb_type>
+     <interconnect>
+       <!--direct name="direct1" input="fle.in" output="ble6.in"/-->
+       <direct name="direct2" input="ble6.out" output="fle.out[0:0]"/>
+       <direct name="direct3" input="fle.clk" output="ble6.clk"/>
+     </interconnect>
+   </mode>
+   <!-- 6-LUT mode definition end -->
+
+Full example can be found at `link
+<https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/master/vtr_flow/arch/timing/k6_frac_N10_40nm.xml>`_.
+
+Validation in packer
+--------------------
+After finishing the architecture description, the next step is to validate that VPR can map logics to each operating modes.
+Since VPR packer will exhaustively try each operating mode and finally map logics to one of it.
+As long as there is an operating mode that is feasible for mapping, VPR will complete packing without errors.
+However, this may shadow the problems for other operating modes.
+It is entirely possible that an operating mode is not defined correctly and is always dropped by VPR during packing.
+Therefore, it is necessary to validate the correctness of each operating mode.
+Here, we will use the syntax ``disable_packing`` which can efficiently reach the goal.
+
+First, we can disable the dual 5-input LUT mode for packer, by changing 
+
+.. code-block:: xml
+
+   <mode name="n2_lut5">
+
+to 
+
+.. code-block:: xml
+
+   <mode name="n2_lut5" disable_packing="true">
+
+As a result, VPR packer will consider the 6-input LUT mode during packing.
+We can try a benchmark `mult_2x2.blif
+<https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/master/vtr_flow/benchmarks/microbenchmarks/mult_2x2.blif>`_
+by following the design flow tutorial :ref:`basic_design_flow_tutorial`.
+If the flow-run succeed, it means that the 6-input LUT mode is functional correct.
+
+Then, we can enable the dual 5-input LUT mode for packer, and disable the 6-input LUT mode, by changing
+
+.. code-block:: xml
+
+   <mode name="n2_lut5" disable_packing="true">
+
+   <mode name="n1_lut6">
+
+to 
+
+.. code-block:: xml
+
+   <mode name="n2_lut5">
+
+   <mode name="n1_lut6" disable_packing="true">
+
+In this case, VPR packer will consider the dual 5-input LUT mode during packing.
+We can again try the same benchmark `mult_2x2.blif
+<https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/master/vtr_flow/benchmarks/microbenchmarks/mult_2x2.blif>`_
+by following the design flow tutorial :ref:`basic_design_flow_tutorial`.
+If the flow-run succeed, it means that the dual 5-input LUT mode is functional correct.
+
+Finally, we validated that both operating modes are functional correct.
```

Comment:
'Finally, after having validated' ... 'that both modes are being successfully used by the packer.'